cmd/goodman/main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/snikch/goodman"
+)
+
+var c chan os.Signal
+var cmds chan *exec.Cmd
+var runners []goodman.Runner
+var hookServerInitalPort = 61322
+var hooksServerCount int
+
+func main() {
+	cmds = make(chan *exec.Cmd, 50)
+	args := os.Args
+	hookPaths := args[1:len(args)]
```

Comment:
Correct. Ignore me. I was more thinking around accessing an index after, which `1` would be - but I forgot about the special case for this that Go does. All good.